common/common.scss

```diff
@@ -65,24 +68,35 @@ pre {
   width: 100%;
 }
 
-.btn,
+.btn:not(.btn-flat),
 .select-kit.dropdown-select-box .dropdown-select-box-header {
   border: 1px solid var(--gf-primary-low-or-low-mid, $primary-low);
   color: var(--primary-high, $primary-high);
   background: transparent;
   border-radius: 4px;
-
-  &:hover {
-    background: var(--tertiary-low);
-    color: var(--tertiary);
+  .d-icon {
+    color: currentColor;
+  }
+  .discourse-no-touch & {
+    &:hover {
+      background: var(--tertiary-low);
+      color: var(--tertiary);
+      .d-icon {
+        color: currentColor;
+      }
+    }
   }
 }
 
+.btn[class*="popup-menu-btn"] {
+  border: none;
```

Comment:
Nitpick! But maybe it's more consistent to make the broder color transparent (that way dimensions don't change)